pkg/edit/vars.go

```diff
@@ -56,6 +55,34 @@ func initVarsAPI(ed *Editor, nb eval.NsBuilder) {
 // ~> echo $b:foo
 // foo
 // ```
+//
+// Because of the above behavior this mechanism is perhaps most useful for adding a module namespace
+// or function to the REPL namespace. For example, you might include this in your
+// `~/.config/elvish/rc.elv` in order to make your Elvish interactive config work on both Windows
+// and UNIX-like systems:
+//
+// ```elvish-transcript
+// use platform
+// if $platform:is-unix {
+//   use unix
+//   edit:add-var unix: $unix:
+// }
+//
+// Similarly you might have functions, such as `$util:ff~` to "find files", that you want to be able
```

Comment:
The "find files" bit in the example doesn't feel very useful. Either an abstract function (just say `f`) or a more common example (like `ll` for `ls -l`) would be more useful.

Reply:
Fair enough, but I use an abbreviation for `ll` to map to `ls -lhtr` so that isn't a good example in my case. I also don't think a more abstract is a good idea. IMHO, it's usually better to provide a concrete example the user might relate to in this situation.